### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JOduMonT**
more or less clear how to integrate the theme into a hugo project
+ not the best way to do it if you already use git for you project

### CODE REVIEW

**Comment by fte378** [comment]
Good change!

File: README.md [hunk 1]
```diff
@@ -21,7 +21,14 @@ See [picturingjordan.com](https://picturingjordan.com) for an example of this th
 - configurable keywords for every post
 
 ## Usage
-Clone the repository to your site's `themes` directory. Refer to [`exampleSite/config.toml`](https://github.com/alanorth/hugo-theme-bootstrap4-blog/blob/master/exampleSite/config.toml) for recommended configuration values.
+Clone the repository to your site's `themes` directory.
+
+### If you already using git
```
Comment (alanorth) [inline README.md:26]: Change section heading to [title case](http://individed.com/code/to-title-case/) and correct "your" to "you're".

File: README.md [hunk 2]
```diff
@@ -21,7 +21,14 @@ See [picturingjordan.com](https://picturingjordan.com) for an example of this th
...
+git submodule add https://github.com/alanorth/hugo-theme-bootstrap4-blog.git themes/hugo-theme-bootstrap4-blog
+hugo-theme-bootstrap4-blog
```
Comment (alanorth) [inline README.md:28]: This looks like a syntax error, is there an extra "hugo-theme-bootstrap4-blog"?

File: README.md [hunk 3]
```diff
@@ -21,7 +21,14 @@ See [picturingjordan.com](https://picturingjordan.com) for an example of this th
...
+
+### Start with
```
Comment (alanorth) [inline README.md:30]: I don't think we need a new section for this. The general case is to clone the theme to your "themes" folder and then to reference the example config. The "If You're Already Using git" section is just an extra case for advanced users and shouldn't change the flow of the documentation.

**Review by alanorth — CHANGES_REQUESTED** [review]
I'm OK with adding the new subsection about cloning as a git submodule (fix the syntax error though). But I don't think we need the new "Start with" section.

**Pull request closed without merging**